form/src/main/res/layout/form_picture.xml

```diff
@@ -6,22 +6,18 @@
     <data>
 
         <import type="android.view.View" />
+        <import type="org.dhis2.form.model.UiEventType"/>
 
         <variable
-            name="label"
-            type="java.lang.String" />
-
-        <variable
-            name="description"
-            type="java.lang.String" />
+            name="item"
+            type="org.dhis2.form.model.FieldUiModel" />
 
     </data>
 
     <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
could we move this layout to form module?

Reply:
I think is already in the form module :smile: